### PR TITLE
Add missing #cobbleworks blocks to cobblegenrandomizer config

### PR DESCRIPTION
**Pull request by Kanzaji**
Resolves #242 missing (actually quite a few) of them 😅 Woopsie

### CODE REVIEW

**Review by MuteTiefling — COMMENTED** [review]
Scoria, no? :D

**Comment by Kanzaji** [comment]
I think scoria was already done 😅 (Tho I can't check right now as my drive with minecraft got corrupted as you know 😢)

File: config/cobblegenrandomizer-common.toml [hunk 1]
```diff
@@ -29,5 +29,5 @@
...
... ["stone", "blue_skies:turquoise_stone", ["blue_skies:turquoise_stone"]], ["cobblestone", "blue_skies:turquoise_cobblestone", ["blue_skies:turquoise_cobblestone"]], ["stone", "blue_skies:rimestone", ["blue_skies:rimestone"]], ["stone", "blue_skies:midnight_sandstone", ["blue_skies:midnight_sandstone"]], ["stone", "blue_skies:lunar_stone", ["blue_skies:lunar_stone"]], ["stone", "blue_skies:crystal_sandstone", ["blue_skies:crystal_sandstone"]], ["stone", "blue_skies:blinding_stone", ["blue_skies:blinding_stone"]], ["stone", "blue_skies:nature_stone", ["blue_skies:nature_stone"]], ["stone", "blue_skies:poison_stone", ["blue_skies:poison_stone"]], ["stone", "byg:windswept_sandstone", ["byg:windswept_sandstone"]], ["stone", "byg:brimstone", ["byg:brimstone"]], ["cobblestone", "byg:cobbled_ether_stone", ["byg:cobbled_ether_stone"]], ["stone", "byg:cryptic_stone", ["byg:cryptic_stone"]], ["cobblestone", "undergarden:tremblecrust", ["undergarden:tremblecrust"]], ["stone", "twilightforest:mazestone", ["twilightforest:mazestone"]], ["stone", "quark:soul_sandstone", ["quark:soul_sandstone"]], ["stone", "create:veridium", ["create:veridium"]], ["stone", "create:scorchia", ["create:scorchia"]]]
```
Comment (MuteTiefling) [inline config/cobblegenrandomizer-common.toml:32]: I mean, it's `scoria` not `scorchia`

Comment (Kanzaji) [reply config/cobblegenrandomizer-common.toml:32]: There is `scoria` and `scorchia` in the modpack tho :sweat_smile:

Comment (MuteTiefling) [reply config/cobblegenrandomizer-common.toml:32]: Oh jeez

**Comment by NielsPilgaard** [comment]
Thanks!

**Pull request merged**